.github/workflows/ci.yaml

```diff
@@ -97,13 +119,45 @@ jobs:
           path: |
             ~/.cargo/
             target/
+            .venv/
           key: ${{ runner.os }}-cargo-${{ hashFiles('**/Cargo.lock') }}
 
       - name: Python Bindings Tests
         run: |
-          just python build
           just python test
 
+  benchmarks:
```

Comment:
I'm not sure where we landed on the "do we run with this on every commit with a small scale factor on every commit and a larger matrix every ~day or n commits?

Reply:
oh I see what happens here! perf.

Reply:
This job currently runs benchmarks on each merge to `main`. We can go ahead and expand on this regarding what we prefer. I just wanted a job that works first.